app/helpers/passport.js

```diff
@@ -19,36 +22,87 @@ module.exports = function(passport) {
     passport.use(new GitHubStrategy({
         clientID: config.authentication.github.clientId,
         clientSecret: config.authentication.github.clientSecret,
-        callbackURL: '/login/callback'
```

Comment:
The callbackURL? It's not hardcoded, I removed it. As of now the callback url is changed in the GitHub application settings.